Source/tracing/ISyslogMonitorClient.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2021 Metrological Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Module.h"
+namespace WPEFramework {
+namespace Logging {
+    class EXTERNAL ISyslogMonitorClient
+    {
+        public:
+            virtual void NotifyLog(const std::string& logmessage) = 0;
```

Comment:
Why a std::string? We use string to be able to move to multi byte codewords in tghe future (e.g. UTF-16).